docs/QSG/docker.md

```diff
@@ -37,91 +37,59 @@ Virtual Docker networking is still in development. The following are known issue
 
+First you will need to install dockerand docker compose, for eaxample
+```
+sudo apt install docker
+sudo apt install docker-compose
+....
+
 Following command could be used to build directly the three images from GitHub:
 
-@@@ as far as can determine the tag doesnt exist	
 ```bash
-$ for image in daos-server daos-admin daos-client ; do \
-	docker build --tag "$image:rocky8.4" \
-		"https://github.com/daos-stack/daos.git#master:utils/docker/vcluster/$image/el8"; \
-  done
+docker build --tag "$image:rocky8.4" \ "https://github.com/daos-stack/daos.git#master:utils/docker/vcluster/$image/el8"
 ```
-@@@Need to add in the ethernet flag above to specify Ethernet adapter `DAOS_IFACE_NAME`
-Note:- `DAOS_IFACE_NAME`: Fabric network interface used by the DAOS engine (default "eth0")
-
 For additional docker, file arguments see <link>
 	
 ## Running the DAOS Containers
-
-### Via Docker Commands  @@section below needs simplification to one path, but as I can’t get this to run yet, I need guidance or a working docker image tag
-
-Once the images are created, the containers could be directly started with docker with the following
-commands:
-
-```bash
-$ export DAOS_IFACE_IP=x.x.x.x
-$ docker run --detach --privileged --name=daos-server --hostname=daos-server \
-	--add-host "daos-server:$DAOS_IFACE_IP" --add-host "daos-admin:$DAOS_IFACE_IP" \
-	--add-host "daos-client:$DAOS_IFACE_IP" --volume=/sys/fs/cgroup:/sys/fs/cgroup:ro \
-	--volume=/dev/hugepages:/dev/hugepages  --tmpfs=/run --network=host \
-	daos-server:rocky8.4
-$ docker run --detach --privileged --name=daos-agent --hostname=daos-agent \
-	--add-host "daos-server:$DAOS_IFACE_IP" --add-host "daos-admin:$DAOS_IFACE_IP" \
-	--add-host "daos-client:$DAOS_IFACE_IP" --volume=/sys/fs/cgroup:/sys/fs/cgroup:ro \
-	--tmpfs=/run --network=host daos-agent:rocky8.4
-$ docker run --detach --privileged --name=daos-client --hostname=daos-client \
-	--add-host "daos-server:$DAOS_IFACE_IP" --add-host "daos-admin:$DAOS_IFACE_IP" \
-	--add-host "daos-client:$DAOS_IFACE_IP" --volume=/sys/fs/cgroup:/sys/fs/cgroup:ro \
-	--tmpfs=/run --network=host daos-client:rocky8.4
-```
-
-The value of the `DAOS_IFACE_IP` shall be replaced with the one of the network interfaces which was
-provided when the images have been built.
-
-Once started, the DAOS server waits for the administrator to format the system.
-This can be done using the following command:
-
-```bash
-$ docker exec daos-admin dmg -i storage format
-```
-
-Upon successful completion of the format, the storage engine is started, and pools
-can be created using the daos admin tool.  For more advanced configurations and usage refer to the
-section [DAOS Tour](https://docs.daos.io/QSG/tour/).
-
-
-### Via docker-compose
-
-From a local tree, a more straightforward way to start the containers could be done with
-`docker-compose` and the following commands:
+From a local tree, a simple way to start the containers is done with a `docker-compose` commands:
```

Comment:
FMHO, it should be more user friendly to use the bash script `daos-cm.sh` instead of `docker-compose.`